CHANGELOG.md

```diff
@@ -12,12 +12,15 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 - Adds a `contentId` prop to the `TemplateAppContainer` component and adds an `id` prop to the `TemplateContent` component. The default value of these `id`s are both set to "mainContent" and it will render as an attribute on the `main` HTML element. This is used as the target for the skip navigation link in consuming applications.
 - Adds an `Accessibility Guide` section to Storybook with a "Skip Navigation" page.
+- Adds an optional key – `accordionType` to the `Accordion`'s `contentData` prop, which allows users to switch the
+  background color of the `Accordion`'s button.
```

Comment:
Move your _new_ changes to the end of the section. Same with the new line in the `### Changes` section.

Reply:
Just to keep chronological order of when changes were merged in.